rakelib/kokoro/kokoro.rb

```diff
@@ -127,6 +131,12 @@ def check_links location_list, base, tail
     broken_links
   end
 
+  def local_docs_test
+    run "bundle exec rake yard"
```

Comment:
I suspect most, if not all, the CI jobs already run yard, so _maybe_ this step won't be necessary?

Reply:
The randomly selected webrisk gem doesn't (at least as far as I can see)